python/ooi_data_explorations/common.py

```diff
@@ -773,12 +773,17 @@ def process_file(catalog_file, gc=False):
         dods_url = 'http://thredds.dataexplorer.oceanobservatories.org/thredds/dodsC/'
     else:
         dods_url = 'https://opendap.oceanobservatories.org/thredds/dodsC/'
+
     url = re.sub('catalog.html\?dataset=', dods_url, catalog_file)
     ds = xr.load_dataset(url + '#fillmismatch', decode_cf=False)
 
     if not ds:
         return None
 
+    # since the CF decoding of the time is failing, reset time here to the correct units and redo
+    ds.time['units'] = 'seconds since 1900-01-01 00:00:00.000Z'
+    ds = xr.decode_cf(ds)
```

Comment:
When you read the url for a given .nc file with decode flag = F, are you worried about this causing oddities down stream for other instrument data types?

Reply:
Yes and no. That is why I immediately below it reset the units for time and then manually run the decode rather than having xarray automagically run it turn loading. Since the time units in the data files are not set correctly from OOINet, xarray cannot decode the time and throws an error `RuntimeError: NetCDF: Access failure`. Here I am just separating the process out into two unique steps, so we don't get the error.